modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.cpp

```diff
@@ -0,0 +1,183 @@
+/******************************************************************************
+ * Copyright 2024 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/drivers/lidar/vanjeelidar/src/vanjeelidar_component.h"
+
+#include <string>
+
+#include <vanjee_driver/driver/input/input.hpp>
```

Comment:
Where are the files? If have source code, you can add in the code

Reply:
we can offer the source code,and pack them as a deb.